README.md

```diff
@@ -2,21 +2,117 @@
 [![Build Status](https://travis-ci.org/thoughtbot/french_toast.svg?branch=master)](https://travis-ci.org/thoughtbot/french_toast)
 [![Code Climate](https://codeclimate.com/github/thoughtbot/french_toast/badges/gpa.svg)](https://codeclimate.com/github/thoughtbot/french_toast)
 
-Communicate information about background jobs.
+Communicate information about background jobs in Rails seamlessly.
+
+FrenchToast attempts to streamline some [progressive enhancement](https://en.wikipedia.org/wiki/Progressive_enhancement) by providing a simple and opinionated interface. In order to best take advantage of this, it is important to render that information on the backend. This allows FrenchToast to figure out the best possible communication stream, by starting at the most enhanced experience and falling back to the most basic.
```

Comment:
I think both are true and are the same thing. Progressive enhancement does build up from the most basic functionality depending on support. The reverse is sort of also true, if they have the support for the most enhanced it will use it, it will fallback to more basic options until finding something they support.